listLatestDocuments.js

```diff
@@ -0,0 +1,60 @@
+// Ignore item #0 and #1 of argv (node bin path and script path)
+const [, , collectionName, fieldsToRetrieve, sortBy, limit = 5] = process.argv;
+
+if (!collectionName || collectionName == "-h" || collectionName == "--help") {
+  console.log(
+    "Get the latest documents in collection. Limit is set to 5 records by default."
+  );
+  console.log(
+    "Usage\t: node getLatestDocuments.js <collectionName> <fieldsToRetrieve> <sortBy> <limit=5>"
```

Comment:
Rename the `getLatestDocuments.js` to `listLatestDocumts.js` to make it consistent with the file name.